### ISSUE TITLE
Documentation suggestions: add warnings about destructive actions

### ISSUE THREAD

**Issue by drbyte** [area17/twill#11]
Hi,
While the documentation describes casually how to install into a `default` (completely uncustomized) Laravel application, it fails to warn how destructive it is to an existing Laravel app.

A few suggestions to include:

1. `twill:install` will replace the app's Exceptions handler

2. `twill:setup` will delete the 2 default `user` and `password_resets` table migrations, even if they've been customized. This also breaks existing migration history.

3. `twill:setup` will also add the `activity_log` table migration from the spatie package, even if such a migration file already exists, resulting in a fatal error during `twill:setup` attempt to run migrations. This aborts setup with no way to recover.

Also, there is a small typo in the docs:
<img width="746" alt="twill docs typo" src="https://user-images.githubusercontent.com/404472/42597520-0295066c-8527-11e8-96e3-0736a3954c76.png">

**Comment by ifox** [comment]
Hi Chris,

This is super helpful, thank you so much for your suggestions.

Twill's documentation site is a Vuepress project and we're going to add it to this repository so that anyone can contribute but I'll make sure to add your suggestions as well as other updates suggested in #2 asap.

We can also think about an install option to avoid those destructive operations in an existing Laravel application.

**Comment by ifox** [comment]
Hi again Chris,

thanks again for your initial feedback! We released [Twill 1.2 ](https://github.com/area17/twill/releases/tag/1.2.0) last week which includes many changes regarding supporting existing applications better, support for L5.7 and a completely revamped getting started documentation. 

1. `twill:install` does not replace the app's Exceptions handler anymore
2. `twill:install` (we got rid of `twilll:setup`, everything happens in a single command now) does not delete Laravel's default migrations anymore
3. That still needs to be fixed, but is most likely a very rare use case. It also raises a more complex question about how an existing activity log can live in tandem with activities created by Twill. I will try to test this out and get back to you here!

Typo has been fixed too, thanks!
